UpSkill_App/UpSkill.Infrastructure/Data/Models/Coach.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using System.Linq;
+
+    public class Coach : ApplicationUser
+    {
+        public IQueryable<Course> Courses { get; set; }
```

Comment:
A Class definition has ICollection<TEntity>. IQueryable<TEntity> is to be the return type of the relevant Repository<TEntity> methods.

Reply:
Thank you for finding my mistake! It has been fixed.